src/main/java/com/buglife/sdk/ScreenshotAnnotatorActivity.java

```
@@ -253,15 +271,20 @@ private void setSelectedTool(Annotation.Type annotationType) {
         mBlurTool.setColorFilter(tintColor);
     }
 
-    private @NonNull Attachment getAttachmentCopyWithAnnotations() {
-        Bitmap output = mAnnotationView.captureDecoratedImage();
-        return mAttachment.getCopy(output);
+    private void saveAnnotatedBitmap() {
+        try {
+            Bitmap output = mAnnotationView.captureDecoratedImage();
+            File file = mAttachment.getFile();
+            output.compress(Bitmap.CompressFormat.PNG, 100, new FileOutputStream(file));
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Swallowed exception! (Sorry 😁)